Documentation/openshift-issues.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Common Issues on OpenShift
+weight: 10620
+indent: true
+---
+
+# Common Issues on OpenShift
+
+## Monitoring Ceph Cluster on OpenShift
+
+OpenShift Console uses OpenShift Prometheus for monitoring and populating data in Storage Dashboard. Following troubleshooting guide assumes that it is allowed to monitor the Ceph Cluster.
```

Comment:
Suggestion: "The OpenShift console uses OpenShift Prometheus for monitoring and populating data in the storage dashboard. Additional configuration is required to monitor the Ceph Cluster from the storage dashboard."